tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -136,27 +136,9 @@ Filtering:
   ExpFilter0:
     Alpha: 36.0
     HalfPower: 210
-    DisableForDebugging: false
+    DisableForDebugging: true
 
 EventsAndTriggers:
-  ? And:
-      - Slabs:
-          EvenlySpaced:
-            Interval: 200
-            Offset: 0
-      # Note: do not adjust step size until a bit of time has passed, because
-      # empirically starting in damped harmonic gauge requires very rapid
-      # grid motion at first.
-      - TimeCompares:
-          Comparison: GreaterThan
-          Value: 0.1
-  : - ChangeSlabSize:
-        DelayChange: 0
-        StepChoosers:
-          - Cfl:
-              SafetyFactor: 0.8
-          - Increase:
-              Factor: 1.2
```

Comment:
Just for my information, why did you remove the change in slab size?

Reply:
I removed it because @geoffrey4444 did;  I just copied his input files that he said that worked.

Reply:
You can revert and squash this change...I was doing fixed time step runs, but I think this adaptive slab chooser is fine for the test input file